CHANGELOG.md

```diff
@@ -1,3 +1,13 @@
+## v0.4.0 [2022-07-21]
+
+_Bug fixes_
+
+- Fixed the `googleworkspace_gmail_message` and the `googleworkspace_gmail_my_message` tables to resolve the plugin validation errors. ([#34](https://github.com/turbot/steampipe-plugin-googleworkspace/pull/34))
```

Comment:
```suggestion
- Fixed inconsistent table names in the `googleworkspace_gmail_message` and the `googleworkspace_gmail_my_message` tables to resolve plugin validation errors. ([#34](https://github.com/turbot/steampipe-plugin-googleworkspace/pull/34))
```

Reply:
The first message is too generic, i.e., what do you mean by Fix the tables to resolve plugin validation errors?